### PR TITLE
Add link to testing dependencies [ci skip]

### LINKED ISSUE #11172: Error on testing development copy
### Description

Getting an error on testing development copy of astropy.

### Expected behavior

There should be no errors or failures.

### Actual behavior

I'm getting following error.
```python
ImportError while importing test module '/home/param/opensource/astropy/astropy/coordinates/tests/test_name_resolve.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../miniconda3/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
astropy/astropy/coordinates/tests/test_name_resolve.py:21: in <module>
    from pytest_remotedata.disable_internet import no_internet
E   ModuleNotFoundError: No module named 'pytest_remotedata'
```

### Steps to Reproduce

1. Clone the repository 
2. `>> cd astropy`
3. `>> pip install -e .`
4. `>> cd ..`
5. `>> pytest`

### System Details

**platform :** 'Linux-4.19.128-microsoft-standard-x86_64-with-glibc2.10'
**sys :**  '3.8.3 (default, May 19 2020, 18:47:26) \n[GCC 7.3.0]'
**numpy :** '1.19.2'
**astropy :** '4.2.dev1241+g883723e05'

### PR DESCRIPTION
**Pull request by parampatidar**
### Description

Fixes #11172

### CODE REVIEW

File: docs/development/workflow/get_devel_version.rst [hunk 1]
```diff
@@ -274,6 +274,9 @@ Testing is an important part of making sure astropy produces reliable,
 reproducible results. Before you try out a new feature or think you have found
 a bug make sure the tests run properly on your system.
 
+before runnig tests, see
+:ref:`testing-dependencies`.
```
Comment (pllim) [inline docs/development/workflow/get_devel_version.rst:278]:
```suggestion
Before running your tests, please see :ref:`testing-dependencies`.
```

**Review by pllim — CHANGES_REQUESTED** [review]
Thanks! Grammatical suggestion below. I also feel like the narrative could be more descriptive.

File: docs/development/workflow/get_devel_version.rst (hunk 1 above)
Comment (parampatidar) [reply docs/development/workflow/get_devel_version.rst]: Thanks for pointing it out. next time I will always be double check for grammatical error or typo.

Comment (parampatidar) [reply docs/development/workflow/get_devel_version.rst]: Thanks for pointing it out. Next time I will always be double check for grammatical error or typo
.

**Review by pllim — APPROVED** [review]
LGTM, thanks!

**Pull request merged**